### PR TITLE
Add validator guide for the testnet

### PR DESCRIPTION
**Pull request by WeHaveCookie**
Didn't be able to test on my own. 
MicrosoftTeams-image

### CODE REVIEW

File: docs/testnet-Validator.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+[Install]
+WantedBy=multi-user.target
+```
+
+> If you need to change port, you can setup with `--prometheus-port` `--rpc-port` and `--ws-port`
+
+then start the service
+```
+sudo systemctl enable tal && sudo service tal start
```
Comment (masterdubs) [inline docs/testnet-Validator.md:104]: rename to galital

File: docs/testnet-Validator.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+```
+wget https://github.com/starkleytech/galital/releases/download/2.0.0/galital && sudo chmod +x ./galital && sudo mv ./galital /usr/bin/galital
+```
+
+## Make the service permanent
+
+create systemd file in /etc/systemd/system/tal.service
+
+```
+sudo nano /etc/systemd/system/tal.service
```
Comment (masterdubs) [inline docs/testnet-Validator.md]: rename to galital

File: docs/testnet-Validator.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+Enable it
+```
+sudo systemctl enable chrony
+```
+
+Allow the process in firewall
+
+```
+sudo apt install firewalld
```
Comment (masterdubs) [inline docs/testnet-Validator.md]: Firewalld isn't default on ubuntu, it's UFW it need to be adapted:
`ufw allow 22 && ufw allow 30333 && ufw enable`

Also always good to install fail2ban:

`apt install -y fail2ban && systemctl enable fail2ban && service fail2ban start `

File: docs/testnet-Validator.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+```
+
+```
+[Unit]
+Description=Galital Validator
+After=network-online.target
+
+[Service]
+
+ExecStart=/usr/bin/galital --port "30333" --name "A Node Name" --validator  --telemetry-url 'wss://substrate-telemetry.starkleytech.com/submit/ 0' --chain galital
```
Comment (masterdubs) [inline docs/testnet-Validator.md]: remove the telemetry-url flag

File: docs/testnet-Validator.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
+# Setup Masternode
```
Comment (masterdubs) [inline docs/testnet-Validator.md]: incorrect title :) 
It's a validator

**Review by masterdubs — CHANGES_REQUESTED** [review]

File: docs/testnet-Validator.md (hunk 5 above)
Comment (WeHaveCookie) [inline docs/testnet-Validator.md]:
```suggestion
# Setup Validator
```

File: docs/testnet-Validator.md (hunk 2 above)
Comment (WeHaveCookie) [reply docs/testnet-Validator.md]:
```suggestion
sudo nano /etc/systemd/system/galital.service
```

File: docs/testnet-Validator.md (hunk 1 above)
Comment (WeHaveCookie) [reply docs/testnet-Validator.md:104]:
```suggestion
sudo systemctl enable galital && sudo service galital start 
```

File: docs/testnet-Validator.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+```
+
+Allow the process in firewall
+
+```
+sudo ufw allow 22
+sudo ufw allow 30333
+sudo ufw enable
+```
+
```
Comment (WeHaveCookie) [inline docs/testnet-Validator.md:68]:
```suggestion

Setup fail2ban
\```
apt install -y fail2ban && systemctl enable fail2ban && service fail2ban start
\```

```

**Pull request merged**